documentation/docs/guides/goose-cli-commands.md

```diff
@@ -383,6 +383,35 @@ After selecting a project, you'll be asked to either:
 - **Start new session**: Start a new session in the selected project
 
 ---
+### web
+
+Start a web-based chat interface for interacting with Goose. This feature provides a lightweight GUI option for CLI users.
+
+**Usage:**
+```bash
+goose web
+```
+
+**Options:**
+- **`-p, --port <PORT>`**: Port number to run the web server on. Default is `3000`.
+- **`--host <HOST>`**: Host to bind the web server to. Default is `127.0.0.1`.
+- **`--open`**: Automatically open the browser when the server starts.
+
+**Examples:**
+```bash
+# Start web interface at `http://127.0.0.1:3000` and open the browser
+goose web --open
+
+# Start web interface at `http://127.0.0.1:8080` 
+goose web --port 8080
+```
+
+:::info
+Use `Ctrl-C` to stop the server.
+:::
+
+---
+
```

Comment:
CLI commands looks great, but I'm thinking this deserves more of a splash. Should we create a new doc for goose web or add it to an existing page so people know what its, why they would use it, and then link to this section on CLI commands?